implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -41,6 +41,7 @@
 @Dependent
 public class JWTAuthContextInfoProvider {
     private static final String AUTHORIZATION_HEADER = "Authorization";
+    private static final String BEARER_SCHEME = "BEARER";
```

Comment:
@acoburn minor one, can you please rename it to `Bearer`, even though the case insensitive comparison is used.

Reply:
My mistake, it should definitely be `Bearer`

Reply:
Np, thanks, I suppose even this all capital case variation can happen in some cases